bundles/org.openhab.binding.pulseaudio/README.md

```diff
@@ -31,7 +31,7 @@ You can use the GUI on the bindings page (click on the pulseaudio binding then "
 
 ```
 binding.pulseaudio:sink=true
-binding.pulseaudio:source=false
+binding.pulseaudio:source=true
```

Comment:
As you set the "source" boolean to the default value "true" in the readme and in the PulseAudioBindingConfiguration file, could you also change it in the binding.xml ?
(also line 23 in the readme, the sink support is not anymore the only one activated by default, you coud mention the source support)